### DISCUSSION TITLE
I need to migrate all data from the hard disk to another one before replacing it with a new one.

### DISCUSSION THREAD

**Discussion by marioaugustorama** [seaweedfs/seaweedfs#5568] [Q&A] [answered]
I have a fully utilized storage, and I need to know if there is a command or proper procedure to migrate all data from one disk so that I can remove it and replace it with a new one. The current disk is showing SMART errors.

**Comment by chrislusf** [comment]
Does the cluster data have replication enabled?

**Reply by marioaugustorama** [reply]
No, the procedure is just for replacement with higher capacity harddisks and subsequent replication.

**Reply by chrislusf** [reply, marked as answer]
How about just copying all files?

**Comment by marioaugustorama** [comment]
Lock volume and copy all data to new disk?

Yes, I had the idea of ​​what we do with LVM!

You are right!
